resources/profiles/Ratrig/machine/RatRig V-Core 3 500.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "RatRig V-Core 3 500",
+    "model_id": "V-Core_3_500",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "RatRig_V-Core",
+    "bed_model": "ratrig-vcore-bed-500.stl",
```

Comment:
bed_model creates z-fighting issues.
Let's avoid using them.

Reply:
The bed model was removed due to a known z bug